### DISCUSSION TITLE
给各位一点建议

### DISCUSSION THREAD

**Discussion by YourComputer011** [mikumifa/biliTickerBuy#272] [聊天专区]
本人为三人三天bw全部拿下
1、开抢时间一定设置为开票时间，晚一秒过验证码的时间会晚1分钟甚至三分钟
2、抢票间隔大胆设置，本人设置为300，所有账号都能正常使用未被冻结
3、请关闭后台其它无关应用，减少占用
4、实操时本人提前20分钟左右准备完毕所有程序和配置文件，提前5分钟左右点击开始抢票
祝各位抢票顺利

**Comment by YourComputer011** [comment]
还有各位一定记得同步一下系统时间，有个朋友的系统时间比实际时间晚了4秒，29号晚上开抢前才排查出来

**Reply by YourComputer011** [reply]
验证码方面，因为我的大号用的手抢，没空过脚本的验证码，所以全都选择的本地过验证码

**Reply by guiding-star** [reply]
是300毫秒吗？

**Reply by wuyaaaaaaaaaaaa** [reply]
问问怎么同步服务器时间

**Reply by peigenr** [reply]
右键任务栏的时间-调整日期/时间-立即同步
（Windows10）

**Reply by washls** [reply]
本地验证码是解压出来就可以使用了吗

**Comment by drinknaixi-1** [comment]
我可以作证，我昨天也三天全抢到了，我和你的流程也基本一致，配置一样

**Comment by fyydvay** [comment]
可以在开抢前30s手动点开始抢票吗

**Comment by mikumifa** [comment]
打算更新一版本直接把延迟变成0

**Reply by melon0918** [reply]
> 打算更新一版本直接把延迟变成0

那样会不会像隔壁BHYG之前一样提前抢了撞风控

**Reply by mikumifa** [reply]
刚刚测试了，好像真不能这样搞、。

**Comment by enguangcai** [comment]
请问如何一个账号抢三天bw普票？是不是要把exe运行三次依次选择12，13，14号

**Comment by MagicToby** [comment]
谢谢大佬，按照配置调试好后4s bml出票

**Comment by lllllis-02** [comment]
应该早点看到的。。。一开二开啥也没抢到，一直报错，还有一个进程完全不开始抢
